DSharpPlus.Commands/Processors/MessageCommands/MessageCommandProcessor.cs

```diff
@@ -21,25 +20,36 @@
 
 namespace DSharpPlus.Commands.Processors.MessageCommands;
 
-public sealed class MessageCommandProcessor : ICommandProcessor<InteractionCreatedEventArgs>
+public sealed class MessageCommandProcessor : ICommandProcessor
 {
-    public IReadOnlyDictionary<Type, ConverterDelegate<InteractionCreatedEventArgs>> Converters => this.slashCommandProcessor?.ConverterDelegates ?? new Dictionary<Type, ConverterDelegate<InteractionCreatedEventArgs>>();
-    private CommandsExtension? extension;
-    private SlashCommandProcessor? slashCommandProcessor;
+    /// <inheritdoc />
+    public Type ContextType => typeof(SlashCommandContext);
 
-    public IReadOnlyList<Command> Commands => this.commands;
-    private List<Command> commands = [];
+    /// <inheritdoc />
+    public IReadOnlyDictionary<Type, IArgumentConverter> Converters =>
+        this.slashCommandProcessor is not null
+            ? Unsafe.As<IReadOnlyDictionary<Type, IArgumentConverter>>(

```

Comment:
why is this necessary?
also, editorconfig violation, should not be expression-bodied